packages/test/package.json

```diff
@@ -5,16 +5,16 @@
   "main": "index.js",
   "private": true,
   "scripts": {
-    "test:integrated": "mocha  --exit dist/cjs/tests/*.js"
+    "test:integrated": "vitest run"
```

Comment:
We should also probably change the `name` to `@microsoft/test-kiota-typescript-libraries`.
at the moment the whole package gets skipped when running build and tests because name doesn't match the filter criteria `"@microsoft/*"`

Reply:
@koros the `test` package is not a Microsoft namespace package, it's only used to run integrated tests. Running it as part of the Lerna workspace commands also introduces unnecessary commands like `build` and `clean`. It also has dependency issues needing IDs and secrets which, during Lerna runtime, we'll have to pass them on the test commands and it's not a recommended practice. As it is, running it as its own step in the CI/CD pipeline is sufficient.